pegasus/sites.v3/code.org/views/teacher_announcements.haml

```diff
@@ -0,0 +1,16 @@
+%div.teacher-announce
+  %a.announce-heading-link{:href=>"http://teacherblog.code.org/"}
+    %div.announce-heading
+      =I18n.t(:dashboard_announcements)
+      %div.heading-view-all
+        =I18n.t(:dashboard_announce_view_all)
+        >>
+  %div.announce-content
+    %div.content-img-container
+      %img.content-img{:src=>"/images/fit-x108/kids4cropped_2x.jpg"}
```

Comment:
So my understanding is that I can constrain the height like above `fit-x108` which seems to have worked. Let me know if another function is better in this case.